test.py

```diff
@@ -132,7 +132,7 @@ def test_build(args, expected=None, unexpected=None):
 check_call('./emsdk activate sdk-tag-1.38.33-64bit')
 
 print('test binaryen source build')
-check_call('./emsdk install --build=Release binaryen-master-64bit')
+check_call('./emsdk install --build=Release --generator=Ninja binaryen-master-64bit')
```

Comment:
It's good to add a test, thanks! But this removes the previous testing, was that intentional? I think we should still test the default configuration.

Reply:
It was and it wasn't. It's a matter of preference I think to use either Makefile or Ninja as the build system in CI, since you control what is installed on the build server, and the resulting build artifacts/tests should be the same whether it's built using Makefiles or Ninja. 

I didn't want to change the default build system to use Ninja, since it's less commonly installed, but I can definitely change it back to use the default Makefiles. I will also remove `ninja-build` from the Dockerfile. I'll do this in a bit. Thanks!

Reply:
Or perhaps using `--generator="Unix Makefiles"` would be better? Test both the `--generator` argument and still use the default Makefile build system?

Reply:
Good idea, yeah, we can test the parameter using Makefiles.